prow/config.yaml

```diff
@@ -270,3 +270,13 @@ periodics:
       # Bazel needs privileged mode in order to sandbox builds.
       securityContext:
         privileged: true
+
+- name: ci-kubernetes-e2e-prow-canary
+  interval: 1h
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e-prow:0.1
+      args:
+      - "--upload=gs://kubernetes-jenkins/pr-logs"
+      - "--git-cache=/root/.cache/git"
```

Comment:
Why aren't these two flags in the image? When will we not want them? Also is --git-cache going to do anything useful in this scenario?